bbq/compiler/function.go

```diff
@@ -21,44 +21,34 @@ package compiler
 import (
 	"math"
 
-	"github.com/onflow/cadence/activations"
 	"github.com/onflow/cadence/errors"
 )
 
 type function[E any] struct {
-	name                string
-	code                []E
-	localCount          uint16
-	locals              *activations.Activations[*local]
-	parameterCount      uint16
-	isCompositeFunction bool
+	name           string
+	code           []E
+	localCount     uint16
+	parameterCount uint16
+	localsDepth    int
```

Comment:
AFAIU, this is not used yet? but will be used once the closures variables are supported in VM?